Dockerfile

```diff
@@ -13,7 +13,10 @@ ARG target_arch
 ENV PATH=/opt/cegcc-${target_arch}/bin:$PATH
 
 RUN apt update && \
-    DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends bison build-essential ca-certificates flex git libgmp-dev libmpc-dev libmpfr-dev texinfo
+    apt upgrade -y && \
```

Comment:
Is there a good reason for running `apt upgrade`? Wouldn't it be more appropriate to use a newer base image version?

Reply:
Oh
I made a mistake.